nativeauthenticator/handlers.py

```diff
@@ -302,12 +355,21 @@ async def post(self):
                 message = "Your password has been changed successfully!"
             else:
                 alert = "alert-danger"
-                pw_len = self.authenticator.minimum_password_length
-                message = (
-                    "Something went wrong! Be sure your new "
-                    f"password has at least {pw_len} characters and is "
-                    "not too common."
-                )
+                minimum_password_length = self.authenticator.minimum_password_length
+                # Error if minimum password length is > 0.
+                if minimum_password_length > 0:
+                    message = (
+                        "Something went wrong!\n"
+                        "Be sure your new password has at least"
+                        f" {minimum_password_length} characters "
+                        "and is not too common."
+                    )
+                # Error if minimum password length is 0.
+                else:
+                    message = (
+                        "Something went wrong!\n"
+                        "Be sure your new password is not too common."
```

Comment:
Are we confident this is due to common password at this point? If so, perhaps we can say that explicitly.

Perhaps elif check_for_common_passwords -> write out this explicitly, then `else` -> write out "Something went wrong"

Reply:
Yes, we are sure. The user enters current password, new password and confirmation. Above this snippet, we deal with wrong current password, wrong confirmation and too short password. Too common password is the only remaining thing it could be. 

I'm happy to make it more explicit, but I think that's better addressed in a PR addressing #73, where I intend to factor out that mechanic entirely, like we discussed.